cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -237,6 +237,8 @@ public static final class AppFabric {
     public static final String PROGRAM_TERMINATOR_INTERVAL_SECS = "app.program.terminator.interval.secs";
     public static final String PROGRAM_TERMINATE_TIME_BUFFER_SECS = "app.program.terminate.time.buffer.secs";
     public static final String PROGRAM_TERMINATOR_TX_BATCH_SIZE = "app.program.terminator.tx.batch.size";
+    public static final String ARTIFACTS_COMPUTE_HASH = "app.artifact.compute.hash";
+    public static final String ARTIFACTS_SNAPSHOT_COMPUTE_HASH = "app.artifact.snapshot.compute.hash";
```

Comment:
nit: let's rename to app.artifact.compute.hash.snapshot so that they appear next to each other when sorted.